ivy/functional/backends/paddle/experimental/set.py

```diff
@@ -0,0 +1,60 @@
+# global
+from typing import Tuple
+import paddle
+from ivy.func_wrapper import with_unsupported_device_and_dtypes
+
+# local
+import ivy.functional.backends.paddle as paddle_backend
+from . import backend_version
+
+
+@with_unsupported_device_and_dtypes(
+    {"2.4.2 and below": {"cpu": ("complex", "int8", "uint8", "int16", "float16")}},
+    backend_version,
+)
+def intersection(
+    x1: paddle.Tensor,
+    x2: paddle.Tensor,
+    /,
+    *,
+    assume_unique: bool = False,
+    return_indices: bool = False,
+) -> Tuple[paddle.Tensor, paddle.Tensor, paddle.Tensor]:
+    x1 = paddle.reshape(x1, [-1])
+    x2 = paddle.reshape(x2, [-1])
+    if not assume_unique:
+        if return_indices:
+            x1, ind1, _, _ = paddle_backend.unique_all(x1)
+            x2, ind2, _, _ = paddle_backend.unique_all(x2)
+        else:
+            x1, _, _, _ = paddle_backend.unique_all(x1)
+            x2, _, _, _ = paddle_backend.unique_all(x2)
```

Comment:
return indices in all cases and ignore them later as they are calculated eitherway

```suggestion
        x1, ind1, _, _ = paddle_backend.unique_all(x1)
        x2, ind2, _, _ = paddle_backend.unique_all(x2)
```